src/miscwrappers.cpp

```diff
@@ -124,7 +124,7 @@ pipe2(int fds[2], int flags)
   }
   return socketpair(AF_UNIX, SOCK_STREAM | newFlags, 0, fds);
 }
-#endif // if (LINUX_VERSION_CODE >= KERNEL_VERSION(2, 6, 27)) &&
+#endif  // if (LINUX_VERSION_CODE >= KERNEL_VERSION(2, 6, 27)) &&
```

Comment:
Same as above.  (In general, you can use github to show you these trivial changes.)